src/Event/Commands/UpdateUiTPASPrices.php

```diff
@@ -27,4 +29,9 @@ public function getTariffs(): Tariffs
     {
         return $this->tariffs;
     }
+
+    public function getPermission(): Permission
+    {
+        return Permission::aanbodBewerken();
+    }
```

Comment:
I added the permission to make sure if a developer would "reuse" the command in an API call it has a permission setup. But now it doesn't make sense because the context is indeed always the AMQP queue. So I will remove the `AuthorizableCommand` interface from the `UpdateUiTPASPrices` command.

Reply:
Okay it doesn't harm of course, was just wondering if it would work because I wasn't sure if the "system user" would have sufficient permissions. But in hindsight it should have all permissions because it does other actions that require permissions as well. So you can also leave it if you want.